lang/c/c.talon

```diff
@@ -25,27 +26,32 @@ settings():
     user.code_public_variable_formatter = "SNAKE_CASE"
 
 # NOTE: migrated from generic, as they were only used here, though once cpp support is added, perhaps these should be migrated to a tag together with the commands below
-state include: insert("#include ")
-state include system: user.insert_between("#include <", ">")
-state include local: user.insert_between('#include "', '"')
+
+include system: user.insert_between("#include <", ">")
+include local: user.insert_between('#include "', '"')
+include <user.code_libraries>:
+    user.code_insert_library(code_libraries, "")
+
 state type deaf: insert("typedef ")
 state type deaf struct:
     insert("typedef struct")
     insert("{\n\n}")
     edit.up()
-    key('tab')
+    key('tab')preprocessor
```

Comment:
```suggestion
    key('tab')
```

Reply:
This may be that you have not finished migration yet but it makes sense for anything where you are doing text navigation in the midst of insertion to migrate to a snippet as this is less likely to fight with builtin editor commands.

Reply:
Apart from the accidental insert of the word "preprocessor", I didn't touch this at all. The use of `typedef` is discouraged in modern C++.

Reply:
Gotcha, OK. If you don't want to touch the existing C support that's fine; I had assumed that your accidental insertion meant you were trying to edit this code.